.ci/test-local-tap.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eo pipefail
+
+. "${0%/*}/libfunctions.sh"
+
+echo '--- :evergreen_tree: Configuring Environment'
+exec 9>&1 # enable tee to write to STDOUT as a file
+test_bottle
```

Comment:
In the production variant, this is run conditionally...
```bash
[[ -n "$BOTTLE" ]] && test_bottle
```
...but in this staging variant, we run it unconditionally.
```bash
test_bottle
```
Should they both be the same? Why or why not?

Reply:
No, in the local test, if you don't have a bottle, you cannot have a tap.

Reply:
I guess you can use the existing release. If fact, unlike the eos package builder steps, you have a release, so you don't even need the local server at all. You can just do the tap.